### PR TITLE
Remove entry for departing staff member

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: app/views/pages/consultants.html.erb [hunk 1]
```diff
@@ -60,7 +29,7 @@
             <td>
               <ul>
                 <li>Research methodology </li>
-                <li> Quantitative analysis </li>
+                <li>Quantitative analysis </li>
                 <li>Political science/Politics </li>
                 <li>Public opinion </li>
```
Comment (christinach) [inline app/views/pages/consultants.html.erb:34]: @kevinreiss I'm approving it. 
I think you should also remove the space at the end of the string. It won't make a difference in the UI from what I see, but why to have extra spaces.
```
<li>Research methodology</li>             
<li>Quantitative analysis</li>
<li>Political science/Politics</li>
<li>Public opinion</li>

**Review by christinach — APPROVED** [review]

File: app/views/pages/consultants.html.erb (hunk 1 above)
Comment (carolyncole) [reply app/views/pages/consultants.html.erb:34]: 👍

**Review by carolyncole — APPROVED** [review]
@kevinreiss do you have time for @christinach's update?  If not just merge it...  I tested it on staging

File: app/views/pages/consultants.html.erb (hunk 1 above)
Comment (kevinreiss) [reply app/views/pages/consultants.html.erb]: Updated this. I think we can merge this now.

**Pull request merged**